release-process.rst

```diff
@@ -256,21 +271,25 @@ is Dec 31, 2026, 24:00 UTC, even if the actual release date slips to Jan 9,
  Feature selection and development
 ***********************************
 
-RFCs have been introduced many years ago and have been proven as being an
-amazing way to avoid conflicts while providing a very good way to propose new
-things to php.net. New features or additions to the core should go through the
-RFC process. It has been done successfully (as the process went well, but the
-features were not necessary accepted) already for a dozen of new features or
-improvements.
+RFCs were introduced many years ago and have proven to be an effective way to
+avoid conflicts while providing a structured process for proposing changes to
+php.net. New features or core additions should go through the RFC process. The
```

Comment:
```suggestion
the PHP programming language. New features or core additions SHOULD go through the RFC process. The
```
?

Reply:
Ah yeah this makes more sense. I actually reword it a bit as that SHOULD was clashing slightly with the paragraph below.